src/test/javascript/spec/component/shared/metis/postings-footer/post-footer/post-footer.component.spec.ts

```diff
@@ -61,4 +92,81 @@ describe('PostFooterComponent', () => {
         const tags = getElements(fixture.debugElement, '.post-tag');
         expect(tags).toHaveLength(metisTags.length);
     });
+
+    it('should be initialized correctly for users that are at least tutors in course', () => {
+        component.posting = post;
+        component.posting.answers = unsortedAnswerArray;
+        metisServiceUserAuthorityStub.mockReturnValue(true);
+        component.ngOnInit();
+        expect(component.isAtLeastTutorInCourse).toEqual(true);
+        expect(component.createdAnswerPost.resolvesPost).toEqual(true);
+    });
+
+    it('should be initialized correctly for users that are not at least tutors in course', () => {
+        component.posting = post;
+        component.posting.answers = unsortedAnswerArray;
+        metisServiceUserAuthorityStub.mockReturnValue(false);
+        component.ngOnInit();
+        expect(component.isAtLeastTutorInCourse).toEqual(false);
```

Comment:
I *suggest* to use either ``toBeFalse()`` and ``toBeTrue()`` everywhere

Reply:
Thanks, incorporated it.